README.md

```diff
@@ -657,17 +657,24 @@ the users understand the source explicitly, although this is not strictly requir
 
 | Artifact Type      | Description                                                                          |
 |--------------------|--------------------------------------------------------------------------------------|
-| `torch.save`       | A model artifact obtained by [Serialized Pickle Object][pytorch-save] (i.e.: `.pt`). |
-| `torch.jit.script` | A model artifact obtained by [`TorchScript`][pytorch-jit-script].                    |
-| `torch.export`     | A model artifact obtained by [`torch.export`][pytorch-export] (i.e.: `.pt2`).        |
-| `torch.compile`    | A model artifact obtained by [`torch.compile`][pytorch-compile].                     |
+| `torch.save`       | A [serialized python pickle object][pytorch-save] (i.e.: `.pt`) which can represent a model or state_dict. |
+| `torch.jit.save` | A [`TorchScript`][pytorch-jit-script] model artifact obtained with one or more of the graph export options Torchscript Tracing and Torchscript Scripting.                    |
+| `torch.export.save`     | A model artifact storing an ExportedProgram obtained by [`torch.export.export`][pytorch-export] (i.e.: `.pt2`).        |
+|  `TFSavedModel`   | A [SavedModel][tf-save] from Tensorflow or Keras. |
+| `Keras_v3`    |  Keras v3 is the [recommended format][keras-recommended] by the Tensorflow team. See this example to [save and load models][keras-example] and the update to date docs [disambiguating different save methods][keras-methods] in TF and Keras..                    |
```

Comment:
I renamed all these to point to specific functions and included an argument in one of them, `tf.keras.Model.export(format='tf_saved_model')` that is tensorflow specific. The other two functions always output the same artifact. Would prefer to use the high level function `tf.keras.Model.export(format='tf_saved_model')` as the artifact type since it is what users will typically interact with rather than a lower level function like `keras.saving.serialize_keras_object`